compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -36,6 +35,12 @@
 #include "llvm/LTO/LTO.h"
 #include "llvm/Bitcode/BitcodeWriter.h"
 
+#if LLVM_VERSION_GE(17, 0)
+#include "llvm/TargetParser/Host.h"
+#else
+#include "llvm/Support/Host.h"
+#endif
```

Comment:
We already include LLVMWrapper.h in this file, so this include shouldn't be necessary. Or alternatively, it should not be part of LLVMWrapper.h (if it's only used in this file).

Reply:
I don't think it's used anywhere, so I'll try to remove them all.

Reply:
Done